### PR TITLE
Torrent weblink

### PR DESCRIPTION
**Pull request by akuma06**

### CODE REVIEW

File: router/upload.go [hunk 1]
```diff
@@ -116,6 +123,12 @@ func (f *UploadForm) ExtractInfo(r *http.Request) error {
 		return ErrInvalidTorrentCategory
 	}
 
+	// WebsiteLink
+	urlRegexp, _ := regexp.Compile(`^(https?:\/\/)?([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w \.-]*)*\/?$`)
```
Comment (sfan5) [inline router/upload.go]: what about IRC links?

Comment (akuma06) [inline router/upload.go]: There was?

Comment (sfan5) [inline router/upload.go]: Yes this was supported on old nyaa

**Comment by akuma06** [comment]
fixed

**Pull request merged**